packages/opencensus-core/src/trace/model/types.ts

```diff
@@ -249,6 +249,51 @@ export interface SpanEventListener {
   onEndSpan(span: RootSpan): void;
 }
 
+/** An event describing a message sent/received between Spans. */
+export enum MessageEventType {
+  /** Unknown event type. */
+  UNSPECIFIED = 0,
```

Comment:
I do like the numeric enums here so on balance I recommend keeping them this way (nice correspondence with the protos too). But on the other hand, use of a string enum could enable the breaking change to be less significant since the strings 'SENT' and 'RECEIVED' would still be valid. Just mentioning it because it's in my mind, but recommend keeping it this way!

Reply:
I compared with other lang. implementation, we use numeric enums for these (```Kind```, ```Link``` and ```MessageEvent```) types. I am trying to get most of the breaking change in this and next release before going to Beta.

Reply:
Makes sense. The sooner the breaking changes come the less the pain in the future.